admin/controller/dashboard/sale.php

```diff
@@ -30,29 +30,26 @@ public function index() {
             $data['percentage'] = 0;
         }
 
-        $sale_total = $this->model_report_sale->getTotalSales();
+        $config_currency = $this->config->get('config_currency');
+
+        $sale_total = $this->currency->format($this->model_report_sale->getTotalSales(), $config_currency, '', false);
 
         if ($sale_total > 1000000000000) {
-            $data['total'] = round($sale_total / 1000000000000, 1);
-            $suffix = 'T';
+            $data['total'] = round($sale_total / 1000000000000, 1) . $this->language->get('trillion_suffix');
         } elseif ($sale_total > 1000000000) {
-            $data['total'] = round($sale_total / 1000000000, 1);
-            $suffix = 'B';
+            $data['total'] = round($sale_total / 1000000000, 1) . $this->language->get('billion_suffix');
         } elseif ($sale_total > 1000000) {
-            $data['total'] = round($sale_total / 1000000, 1);
-            $suffix = 'M';
+            $data['total'] = round($sale_total / 1000000, 1) . $this->language->get('million_suffix');
         } elseif ($sale_total > 1000) {
-            $data['total'] = round($sale_total / 1000, 1);
-            $suffix = 'K';
+            $data['total'] = round($sale_total / 1000, 1) . $this->language->get('thousand_suffix');
         } else {
             $data['total'] = round($sale_total);
-            $suffix = '';
         }
 
-        $data['total'] = $this->currency->format($data['total'], $this->config->get('config_currency')) . $suffix;
+        $data['total'] = $config_currency . ' ' . $data['total'];
```

Comment:
This may not be good for some countries (i.e. Turkey as local merchants don't know its **code**=TRY but its **right symbol**=TL or new **left symbol**=₺) and RTL languages. I would suggest to add a new variable, $point = true, to the Currency=>format function and check in line 87 and 93. What do you think?